src/main/java/cceclipseplugin/core/EclipseRequestManager.java

```diff
@@ -116,7 +114,8 @@ public void pullFileAndCreate(IProject p, Project ccp, File file, IProgressMonit
 							IFolder newFolder = p.getFolder(currentFolder);
 							try {
 								if (!newFolder.exists()) {
-									pm.putFileInWarnList(relPath.toString(), FileCreateNotification.class);
+									IPath workspaceRelativePath = p.getFullPath().append(relPath);
```

Comment:
Make sure workspaceRelativePath has the filename in it; line 104 is probably where this should be done